static-assets/scripts/host.js

```diff
@@ -73,7 +73,19 @@
   });
 
   communicator.subscribe(Topics.ICE_ZONE_ON, function (message, scope) {
+    var subscribeCallback = function (_message) {
+      switch (_message.type) {
+        case "FORM_RDY": {
```

Comment:
Spell it fully (FORM_READY). Also, let's be more specific: `FORM_ENGINE_RENDER_COMPLETE`